docs/ops.html

```diff
@@ -1815,6 +1815,162 @@ <h4 class="anchor-heading"><a id="remote_jmx" class="anchor-link"></a><a href="#
       </tr>
   </tbody></table>
 
+<h5 class="anchor-heading"><a id="kraft_monitoring" class="anchor-link"></a><a href="#kraft_monitoring">KRaft Metrics</a></h5>
+All of the following metrics allow monitoring of the KRaft quourm and 
```

Comment:
and [what?]

Reply:
Hmm.! how did that get deleted. Fixing it. Thanks for catching.